google/cloud/security/common/data_access/sql_queries/create_tables.py

```diff
@@ -73,3 +73,12 @@
         PRIMARY KEY (`id`)
     ) ENGINE=InnoDB DEFAULT CHARSET=utf8;
 """
+
+CREATE_GSUITE_ACCOUNT_GROUPS_TABLE = """
+    CREATE TABLE `{0}` (
+        `id` bigint(20) unsigned NOT NULL AUTO_INCREMENT,
```

Comment:
[Response](https://developers.google.com/admin-sdk/directory/v1/guides/manage-groups#get_all_domain_groups)

Right now I'm just collect group_id and group_email.